massa-execution-worker/src/execution.rs

```diff
@@ -2189,27 +2191,53 @@ impl ExecutionState {
         )
     }
 
+    pub fn get_final_datastore_keys(
+        &self,
+        addr: &Address,
+        prefix: &[u8],
+        offset: Option<Bound<Vec<u8>>>,
```

Comment:
do not use `Option<Bound>`, use the variant `Bound::Unbounded` when we don't want a bound